common/v2/components/NetworkSelectDropdown.tsx

```diff
@@ -32,12 +63,17 @@ function NetworkSelectDropdown({ network, accountType, onChange }: Props) {
   return (
     <div>
       <label>{translate('SELECT_NETWORK_LABEL')}</label>
-      <ComboBox
-        value={network}
-        items={new Set(validNetworks.sort())}
-        placeholder={DEFAULT_NETWORK}
-        onChange={({ target: { value } }) => onChange(value as NetworkId)}
-      />
+      <DropdownContainer>
+        <Dropdown
+          value={{ label: network }}
+          options={validNetworks.sort().map(n => ({ value: n, label: n }))}
```

Comment:
From what I understand this forces us to use the `name` as a `NetworkId`.
How about changing `validNetworks` to return an object `{ label: n.name, value: n }`
It should allow us on L71 to do `onChange(value.id)`

Reply:
Good call, thanks!